lib/paths/path.js

```diff
@@ -246,6 +246,27 @@ class Path {
 		return responseHeaders;
 	}
 
+	findPreferredExampleByRequest(httpRequestBody) {
+
+		if(this.requestBody.content) {
+			let resultingExampleName;
+
+			Object.entries(this.requestBody.content).find(contentEntries => {
```

Comment:
I think in some places this project assumes `application/json` as the content type, if we're comfortable with that assumption then I think I can make this a little more simple here.